src/main/java/com/iridium/iridiumskyblock/configs/Messages.java

```diff
@@ -154,6 +154,7 @@
     public String islandChatEnabled = "%prefix% &7Island Chat Enabled!";
     public String islandChatDisabled = "%prefix% &7Island Chat Disabled!";
     public String borderColorDisabled = "%prefix% &7That border color has been disabled.";
+    public String activeCooldown = "%prefix% &7You cannot do that due to a running cooldown, please wait %duration%!";
```

Comment:
feel like it would make more sence just to merge this and the duration format?

Reply:
Possible but we also have the time format in the config.yml

Reply:
Ye but the time in config.yml is for multiple stuff and I just feel like it makes more sence to include this into active cooldown